spec/rubocop/cop/rspec/empty_line_after_final_let_spec.rb

```diff
@@ -185,6 +200,31 @@
   end
   RUBY
 
+  include_examples 'autocorrect',
+                   bad_example,
+                   good_example
+
+  bad_example = <<-RUBY
+  RSpec.describe User do
+    let(:params) { <<-DOC }
+     I'm super annoying!
```

Comment:
The truth about heredocs.

Reply:
Actually, perhaps `parser` and `ast` gems could be changed to provide better support for heredocs and "line continuation string"…

Reply:
> "line continuation string"

WDYM by that?

Reply:
Oh, do things like

```ruby
'hi i'm a string'\
'and still continuing'
```

produce weird locations too?

Reply:
Yup. I will open an issue on Rubocop one of these days.